javascript-loops/main.js

```diff
@@ -0,0 +1,68 @@
+/* exported getNumbersToTen,
+            getEvenNumbersToTwenty,
+            repeatWord,
+            logEachCharacter,
+            doubleAll,
+            getKeys,
+            getValues
+ */
+
+function getNumbersToTen() {
+  var numbers = [];
+  var currentNumber = 0;
+  while (currentNumber < 10) {
+    currentNumber++;
+    numbers.push(currentNumber);
+  }
+  return numbers;
+}
+
+function getEvenNumbersToTwenty() {
+  var evenNumbers = [];
+  var currentNumber = 2;
+  while (currentNumber <= 20) {
+    evenNumbers.push(currentNumber);
+    currentNumber += 2;
+  }
+  return evenNumbers;
+}
+
+function repeatWord(word, times) {
+  var count = 1;
+  var repeted = '';
+  while (count < times) {
+    count++;
+    repeted += word;
+  }
+  return repeted;
+}
+
+function logEachCharacter(string) {
+  for (var i = 0; i < string.length; i++) {
+    console.log(string[i]);
+  }
+}
+
+function doubleAll(numbers) {
+  var doubled = [];
+  for (var i = 0; i < numbers.length; i++) {
+    doubled.push(numbers[i] * 2);
+  }
+  return doubled;
+}
+
+function getKeys(object) {
+  var keys = [];
+  for (var key in object) {
+    keys.push(`${key}`);
+  }
+  return keys;
+}
+
+function getValues(object) {
+  var values = [];
+  for (var value in object) {
+    values.push(`${object[value]}`);
+  }
+  return values;
+}
```

Comment:
This should return an array of the actual values, not an array of their string versions:

---

![image](https://user-images.githubusercontent.com/16143771/145318063-8ea768f3-cac4-4bc0-b270-8d383d5a9010.png)